json-schemas/src/account-stats.json

```diff
@@ -339,6 +359,11 @@
           "inclusiveMinimum": 0,
           "description": "Total outbound realtime message count (sent from the Ably service to clients), excluding presence messages."
         },
+        "messages.outbound.realtime.messages.billableCount": {
+          "type": "number",
+          "inclusiveMinimum": 0,
+          "description": "Total number of outbound realtime messages that are billable (sent from the Ably service to clients), excluding presence messages."
```

Comment:
does this exclude state messages too?

Reply:
do state messages get counted in `messages.outbound.realtime.messages.count`?

Reply:
no they have their own key separate from regular messages, exactly like presence